qemu_v8.mk

```diff
@@ -29,14 +29,14 @@ GICV3 = y
 QEMU_VIRTFS_AUTOMOUNT = y
 endif
 
+include common.mk
+
```

Comment:
Please leave the include where it is and replace `$(UNAME_M)` below with `$(shell uname -m)`. Reason being, `common.mk` uses `$(RUST_ENABLE)`.

Reply:
good catch. (I thought the rust build was OK, but actually you're right, the rust examples still not built after checking the build logs)

I delete the "patch-1" branch by mistake just now, which resulted this PR invalid... I will go create a new PR, thanks.